libs/postgres_ffi/src/xlog_utils.rs

```diff
@@ -119,11 +119,6 @@ pub fn generate_pg_control(
     // Generate new pg_control needed for bootstrap
     checkpoint.redo = normalize_lsn(lsn, WAL_SEGMENT_SIZE).0;
 
-    //reset some fields we don't want to preserve
-    //TODO Check this.
-    //We may need to determine the value from twophase data.
-    checkpoint.oldestActiveXid = 0;
-
```

Comment:
Do we store correct and track the oldestActiveXid value correctly in the pageserver now? I think not. Looking at the code in walingest.rs, we ignore all other fields from checkpoint records except oldestXid.

Reply:
Add tracking also of `oldestActriveXid`

Reply:
Ok, please add a test that fails when you remove the tracking.

Reply:
If I do not track `oldestActiveXid`, then assertion failure will happen in xlog.c:
```
			if (wasShutdown)
				oldestActiveXID = PrescanPreparedTransactions(&xids, &nxids);
			else
				oldestActiveXID = checkPoint.oldestActiveXid;
			Assert(TransactionIdIsValid(oldestActiveXID));
```
if I assign FirstNormalTransactionId to checkPoint.oldestActiveXid, then it is most conservative estimation of oldest xid and so it can not cause any errors. Well, it is not true in case of xid  wraparound (xid > 2^31).  But it is hard to cause wraparound in the test.